autolab/docker
# Updateable Autolab config management

This allows Autolab configs to be able to be updated, by specifying the file mappings.

I also moved setup.sh to become a Makefile, which allows users (or us) to create/clear the configs easily

Testing:
1. `cd autolab`, then `git checkout docker_env_configs`
2. `cd ..` back to docker repo directory
3. Run `make clean`
4. Run `make`
5. Run `docker-compose build`
6. Run `docker-compose up`

You should be able to see Autolab on localhost:80 now (nowithstanding the db issue)
